trajectory_publisher:
  rovers: ['RR01', 'RR04']
  dt: .02
  trajectory_file: '/home/swarm/data/trajectories/two_rover.json'
  is_cyclic: True